models/marketo.yml

```diff
@@ -37,9 +37,6 @@ models:
       - name: workspace_name
         description: Name of the parent workspace, if applicable.
 
-      - name: count_sends
```

Comment:
why are we removing count_sends btw?

Reply:
Good call! @sisu-callum called this out because `persist_docs` was failing his runs because `email_sends` did not exist in the final model and did in the yml. 

@sisu-callum do your runs fail if we only remove the `email_sends` description for the `marketo__email_sends` table? It looks like that is the only table where email_sends is included in the yml and not the final table.

Reply:
I removed count_sends because it wasn't being referenced in any of the models. It looks like it might have been a holdover from an older version. 

@fivetran-joemarkiewicz I'm realizing now that we can actually leave this in if we're enforcing `0.20.0` in this package, given the `IF_EXISTS` fix for Snowflake `persist_docs`